### PR TITLE
Modal: set content width to 100% to prevent overflow bug in IE

### PR DESCRIPTION
**Pull request by anniejiwon**

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 4b535efd3617a052879323c00fa5b632d6dfd9d1

https://deploy-preview-315--gestalt.netlify.com

**Comment by christianvuerings** [comment]
@anniejiwon could you provide a before / after screenshot?

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :file_folder: Bundlesize - <i>Bundle size increase detected, please review</i>
  </td>
    </tr>
  </tbody>
</table>


Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
gestalt.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 267.44 KB | 267.48 KB | 59.93 KB | 59.92 KB
gestalt.es.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 264.75 KB | 264.78 KB | 59.56 KB | 59.55 KB
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gestalt/src/Modal.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ export default class Modal extends React.Component<Props> {
                     position="relative"
                     display="flex"
                     direction="column"
+                    width="100%"
```
Comment (pfarejowicz) [inline packages/gestalt/src/Modal.js]: `column={12}` is better since its a class and not an inline style

**Comment by anniejiwon** [comment]
screen shot 2018-08-27 at 10 31 16 am
screen shot 2018-08-27 at 10 28 13 am

**Pull request merged**